deploy/runtools/run_farm.py

```diff
@@ -140,11 +142,12 @@ class RunFarm(metaclass=abc.ABCMeta):
     mapper_consumed: Dict[str, int]
 
     default_simulation_dir: str
+    metasimulation_enabled: bool
 
     def __init__(self, args: Dict[str, Any], metasimulation_enabled: bool) -> None:
         self.args = args
         self.metasimulation_enabled = metasimulation_enabled
-        self.default_simulation_dir = self.args.get("default_simulation_dir")
+        self.default_simulation_dir = self.args.get("default_simulation_dir", "/home/centos")
```

Comment:
IMO this shouldn't have a default, run farms should specify the default directory themselves. But in the interest of time, this should go in the documentation that this is a default.